### DISCUSSION TITLE
How to list out the info of the slices in the reports?

### DISCUSSION THREAD

**Discussion by jeanneyue** [insightsengineering/teal#1279] [Q&A]
Hi,

Can the selected values of the slices be listed in the reports?

Best,
Jeanne

**Comment by donyunardi** [comment]
Hi @jeanneyue 

Sorry for the late reply. Yes, you can list the selected values in the slices within the report. Most, if not all, of our `teal` modules from `teal.modules.general/clinical` already include the selected values from filters in the reporter feature, so I assume you’re creating your own custom module and want to add the filter values to your reporter.

I recently did a virtual workshop on how you can add those values in your reporter when creating a custom teal module:
https://youtu.be/WnTgIWU59RM?feature=shared&t=2782

There's also [this vignette](https://insightsengineering.github.io/teal.slice/latest-tag/articles/filter-panel-for-developers.html) if you want to learn more about the Filter Panel API.

Hope this helps but let me know if you need further clarifications.
